pkg/operator/ceph/cluster/osd/pod.go

```diff
@@ -152,18 +152,29 @@ func (c *Cluster) makeDeployment(nodeName string, devices []rookalpha.Device, se
 	if !osd.IsDirectory && osd.IsFileStore {
```

Comment:
For the mon and mgr, the terminology I followed was to call the pod/container specification file `spec.go` (and `spec_test.go`), which seemed to be something semi-consistent throughout the code. I think it would be helpful to use the same terminology for the same/similar pieces of the various daemons. It feels like it makes it easier to discuss them and grok them easier. Do you think it would be good to rename this to `spec.go`, or on the flip side, to change the mon and mgr `spec.go` to `pod.go`? The OSD is also a complex component; it might be appropriate to have a `spec.go` *and* a `pod.go` (or `container.go`, or something else) to break up the complexity into smaller bits if that feels more right to you.

Reply:
renaming it to `spec.go` seems reasonable, i'll go ahead and make that change with this PR.